docs/content/Configuration/Environment-Variables-Reference.md

```diff
@@ -45,10 +45,10 @@ options][link-config].
 | `CUBEJS_CUBESTORE_HOST`         | The hostname of the Cube Store deployment                                                                                         | A valid hostname                                                                |
 | `CUBEJS_CUBESTORE_PORT`         | The port of the Cube Store deployment                                                                                             | A valid port number                                                             |
 | `CUBEJS_EXT_DB_TYPE`            | [Alternative to Cube Store storage for pre-aggregations][link-preaggregations-storage]                           | A database type supported by Cube.js
-| `CUBEJS_EXT_DB_HOST`            | The host URL for an external pre-aggregations database                                                           | A valid database host URL                                     
```

Comment:
I wonder if all of these `CUBEJS_EXT_DB_*`should really say "...for an *alternative* pre-aggregations database" or "...for a pre-aggregations database other than Cube Store" to emphasize the existence of `CUBEJS_CUBESTORE_*`.